cmd/surf/services/tests/parallelfilesprocessor_test.go

```diff
@@ -4,8 +4,8 @@ import (
 	"context"
 	"errors"
 	"fmt"
-	"github.com/CloudHub360/ch360.go/cmd/surf/commands"
-	"github.com/CloudHub360/ch360.go/cmd/surf/commands/mocks"
+	"github.com/CloudHub360/ch360.go/cmd/surf/services"
+	mocks2 "github.com/CloudHub360/ch360.go/cmd/surf/services/mocks"
```

Comment:
```suggestion
	"github.com/CloudHub360/ch360.go/cmd/surf/services/mocks"
```
I can't see why this is aliased to `mocks2`?

Reply:
Good spot.